zipline/finance/controls.py

```diff
@@ -425,3 +430,40 @@ def validate(self,
         """
         if _account.leverage > self.max_leverage:
             self.fail()
+
+
+class MinLeverage(AccountControl):
+    """
+    AccountControl representing a limit on the minimum leverage allowed
+    by the algorithm after a threshold period of time.
+    """
+
+    @expect_types(
+        __funcname='MinLeverage',
+        min_leverage=(int, float),
+        deadline=pd.datetime
```

Comment:
`pd.datetime` is just an alias of the standard library `datetime.datetime` (presumably b/c something in the top level pandas module does `from datetime import datetime`).  We should use the standard library location here.